cli/spec/kontena/cli/stacks/labels/remove_command_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'kontena/cli/stacks/label_command'
+require 'kontena/cli/stacks/labels/remove_command'
+
+describe Kontena::Cli::Stacks::Labels::RemoveCommand do
+  include ClientHelpers
+  include OutputHelpers
+
+  # stack name
+  let :name do
+    'test-stack'
+  end
+
+  # shared endpoint - the same for all cases
+  let :endpoint do
+    "stacks/test-grid/#{name}"
+  end
+
+  # scratch stack definition - use it as is, or modify to accomodate specific test case
+  let :scratch do
```

Comment:
I think it's cleaner if the base form is `:stack` and then the modified versions in different contexts are something like `let(:stack_with_labels) { stack.merge('labels' => %w(foo=bar xyz=zyx)) }`

Reply:
✅ RESOLVED